.travis.yml

```diff
@@ -27,7 +27,7 @@ matrix:
       env: TASK=code-lint
       # Running Code Linter -- Requires @loopback/build so it's bootstrapped
       script:
-        - lerna bootstrap --scope @loopback/build --scope @loopback/tslint-config
+        - lerna bootstrap  --scope @loopback/tslint-config --scope @loopback/eslint-config --scope @loopback/build
```

Comment:
I thought we don't want to enable eslint-based linting yet, at least not as part of this PR. In which case I think this line should be reverted. Thoughts?

Reply:
Reverted

Reply:
I had to add it back so that CI works.